cirq-google/cirq_google/devices/grid_device.py

```diff
@@ -38,7 +38,7 @@
 from cirq_google.api import v2
 from cirq_google.devices import known_devices
 from cirq_google.experimental import ops as experimental_ops
-from cirq_google.ops.coupler import Coupler
+from cirq_google import ops as cg_ops
```

Comment:
Let's then delete line 36 and only use `cg_ops` for that module.

Reply:
Ah, missed that this was cirq_google.ops (thought it was cirq.ops).  Reverted this and changed to ops.Coupler for brevity.